src/core/execute.c

```diff
@@ -730,14 +730,20 @@ static int restore_confirm_stdio(int *saved_stdin, int *saved_stdout) {
         return r;
 }
 
-static int ask_for_confirmation(const char *vc, const char *cmdline, char *response) {
+static int ask_for_confirmation(const char *vc, const Unit *u, const char *cmdline, char *response) {
         int saved_stdout = -1, saved_stdin = -1, r;
 
         r = setup_confirm_stdio(vc, &saved_stdin, &saved_stdout);
         if (r < 0)
                 return write_confirm_error(r, vc);
 
-        r = ask_char(response, "yns", "Execute %s? [Yes, No, Skip] ", cmdline);
+ask_again:
+        r = ask_char(response, "ynvs", "Start %s ? [Yes, No, View, Skip] ", u->id);
```

Comment:
Yeah, this patch makes the whole thing much more usable.

Couple nitpicks:
There's a spurious space between in "%s ?".

I don't think you can change "Execute" to "Start". The problem is that systemd first prints a message about starting a service, and then asks the question (I guess this depends on the unit Type= value):
```
[  OK  ] Started Console Getty.
Start console-getty.service ? [Yes, No, View, Skip]
```

If a question for a unit with a shorter name is asked after a question for a shorter unit, with get visual garbage:
```
Start dbus.service ? [Yes, No, View, Skip] iew, Skip]  
```
I'm not sure if a) this is worth fixing, and b) what the best solution would be. You could always print a newline after the answer, or maybe it'd be better to use `ANSI_ERASE_TO_END_OF_LINE`? We use that in status_vprintf and it seems to work nicely.